src/lib/models/outoforder/Core.cc

```diff
@@ -223,14 +224,9 @@ std::map<std::string, std::string> Core::getStats() const {
   auto backendStalls = dispatchIssueUnit_.getBackendStalls();
   auto portBusyStalls = dispatchIssueUnit_.getPortBusyStalls();
 
-  uint64_t totalBranchesExecuted = 0;
-  uint64_t totalBranchMispredicts = 0;
+  uint64_t totalBranchesExecuted = fetchUnit_.getBranchFetchedCount();
```

Comment:
This doesn't seem correct to me. Just because a branch is fetched doesn't mean it will have been executed. Either we should re-define this statistic, or update how it is collected (i.e. either from the EUs by storing how many branch instructions execute, or from the ROB by probing if branch instructions have executed on flush or commit)

Reply:
Yes, this was me only half updating the language used.  We should be using instructions fetched.  It is branches encountered by the branch predictor (0x12) and branch mispredicts (0x10) that are the PMU events that real-hardware misprediction stats are based on.  I've updated the language to fetched throughout.